src/submit_jobs.py

```diff
@@ -1,12 +1,15 @@
 import argparse
+import os
 import time
 from datetime import datetime, timedelta
 
 from joblib import Parallel, delayed
+
```

Comment:
**[Nitpick, non-blocking]** I think this is a stray newline, yeah?

```suggestion
```

Reply:
Ruff put that in there I think. There's supposed to be a separating between third-party and first-party module imports.

Reply:
My bad, that was obvious and I have no idea why I missed it 🤦🏻‍♀️